From 4d644b9f69f503d4b3c93826cb5a9f8555517d52 Mon Sep 17 00:00:00 2001
From: Sergii Piatakov <sergii.piatakov@globallogic.com>
Date: Thu, 15 Nov 2018 15:26:35 +0200
Subject: [PATCH] formatting: use tabs instead of spaces

Signed-off-by: Sergii Piatakov <sergii.piatakov@globallogic.com>
---
 calculator.cpp | 5 ++---
 calculator.h   | 6 +++---
 2 files changed, 5 insertions(+), 6 deletions(-)

diff --git a/calculator.cpp b/calculator.cpp
index 61e22eb..e267ebb 100644
--- a/calculator.cpp
+++ b/calculator.cpp
@@ -2,11 +2,10 @@
 
 int Calculator::Add (double a, double b)
 {
-    return a + b + 0.5;
-
+        return a + b + 0.5;
 }
 
 int Calculator::Sub (double a, double b)
 {
-    return Add (a, -b);
+	return Add (a, -b);
 }
diff --git a/calculator.h b/calculator.h
index 3740907..d59d596 100644
--- a/calculator.h
+++ b/calculator.h
@@ -3,9 +3,9 @@
 
 class Calculator
 {
-    public:
-        int Add (double, double);
-        int Sub (double, double);
+	public:
+		int Add (double, double);
+		int Sub (double, double);
 };
 
 #endif//CALCULATOR_H
-- 
2.32.1 (Apple Git-133)

